src/main/java/pl/llp/aircasting/screens/common/ToggleAircastingManager.java

```diff
@@ -69,7 +79,7 @@ private void stopMobileAirCasting(Session session) {
         } else {
             currentSessionManager.stopSession();
 
-            if(session.isLocationless()) {
+            if (session.isLocationless()) {
                 currentSessionManager.finishSession(sessionId, false);
```

Comment:
I see `finishSession` is using a bool param. You may want to take a look at https://github.com/troessner/reek/blob/master/docs/Boolean-Parameter.md

Reply:
Nice, helpful stuff! However here the path of further execution is not determined by that argument - it's just a value being passed to another method, to set it on a session. This could probably be done better, though.